openssl-sys/src/obj_mac.rs

```diff
@@ -996,3 +996,5 @@ pub const NID_sha3_512: c_int = 1034;
 pub const NID_shake128: c_int = 1100;
 #[cfg(ossl111)]
 pub const NID_shake256: c_int = 1101;
+#[cfg(ossl110)]
+pub const NID_chacha20_poly1305: c_int = 1018;
```

Comment:
Please add

```
#[cfg(libressl271)]
pub const NID_chacha20_poly1305: c_int = 967;
```

https://github.com/openbsd/src/blob/master/lib/libcrypto/objects/obj_mac.num#L967

Reply:
Should I add boringssl too then?

https://github.com/google/boringssl/blob/f714cb2935906a2f085c3b89d7e206af94627b56/include/openssl/nid.h#L4204

Reply:
Done.